src/test/java/com/microsoft/azure/servicebus/jms/connectionString/QueueTests.java

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) Microsoft. All rights reserved.
+package com.microsoft.azure.servicebus.jms.connectionString;
```

Comment:
In Java convection it is better to avoid camel case naming for packages.
connectionstring or connection.string